cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js

```diff
@@ -73,21 +67,17 @@ export default ({ service, variant, pageType, path }) => {
     getPrivacyBanner(service, variant).should('be.visible');
     getCookieBannerCanonical(service, variant).should('not.exist');
 
-    assertCookieValues({
-      ckns_privacy: 'july2019',
-      ckns_policy: '000',
-    });
+    assertCookieHasValue('ckns_privacy', 'july2019');
+    assertCookieHasValue('ckns_policy', '000');
 
     getPrivacyBannerAccept(service, variant).click();
     getCookieBannerRejectCanonical(service, variant).click();
 
     visitPage(path, pageType);
 
-    assertCookieValues({
-      ckns_explicit: '1',
-      ckns_privacy: 'july2019',
-      ckns_policy: '000',
-    });
+    assertCookieHasOneOfValues('ckns_explicit', ['1', '2']);
```

Comment:
I think this looks good. Only thing I thought that could be added is a comment to explain the reason that we check 1 or 2 is because of the value being set differently in the UK and Europe, so future people working on this can understand better.

Reply:
Good suggestion. Hopefully this is clearer https://github.com/bbc/simorgh/pull/9398/commits/cfdac7b82f085672cb1c66608065296ec5e72d5a